src/store/actions/deposit.js

```diff
@@ -16,8 +17,10 @@ import { CALLBACKS } from "../../utils/callbacks";
  * @param {string} total_deposit -  The amount to deposit
  */
 export const createDeposit = params => async (dispatch, getState, lh) => {
-  const { amount, partner, channelId, tokenAddress } = params;
+  const { partner, channelId, tokenAddress } = params;
+  let { amount } = params;
   const channel = getChannelByIdAndToken(channelId, tokenAddress);
+  amount = new BigNumber(amount).plus(channel.total_deposit).toString();
```

Comment:
We need to add the offChainBalance instead of total deposit, and make sure that the deposit fails when the user makes a new deposit that is lower than the previous total deposit